### PR TITLE
chore: update core Angular and React samples to version 19

### PR DESCRIPTION
**Pull request by andygup**
This PR updates the respective @arcgis/core samples.

/cc @omarkawach @manugadde @jimmychen97

### CODE REVIEW

File: core-samples/jsapi-angular-cli/tsconfig.json [hunk 1]
```diff
@@ -1,28 +1,22 @@
...
-    "sourceMap": true,
-    "declaration": false,
-    "downlevelIteration": true,
     "experimentalDecorators": true,
     "moduleResolution": "bundler",
     "importHelpers": true,
     "target": "ES2022",
-    "module": "ES2022",
-    "useDefineForClassFields": false,
-    "lib": [
```
Comment (dasa) [inline core-samples/jsapi-angular-cli/tsconfig.json:22]: Removing `lib` seems odd.

**Review by dasa — APPROVED** [review]

**Review by odoe — APPROVED** [review]
React one looks good!

**Review by sagewall — APPROVED** [review]
LGTM, Seeing a weird stencil related warning when running `npm run watch` or `npm run build` in the angular sample, but that's likely not directly sample related.
```
Application bundle generation complete. [2.353 seconds]

▲ [WARNING] The glob pattern import("./**/*.entry.js*") did not match any files [empty-glob]

    node_modules/@stencil/core/internal/client/index.js:82:2:
      82 │   `./${bundleId}.entry.js${BUILD4.hotModuleReplacement && hmrVersi...
         ╵   ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~


Output location: C:\Users\sag11876\Code\github\jsapi-resources\core-samples\jsapi-angular-cli\dist
```

File: core-samples/jsapi-angular-cli/tsconfig.json (hunk 1 above)
Comment (andygup) [reply core-samples/jsapi-angular-cli/tsconfig.json:22]: I thought it was odd, as well. It wasn't there when I used the angular CLI to create the project. Should I add it back?

Comment (dasa) [reply core-samples/jsapi-angular-cli/tsconfig.json:22]: OK, I'd leave it out then to match the CLI.

**Comment by andygup** [comment]
> LGTM, Seeing a weird stencil related warning when running npm run watch or npm run build in the angular sample

@sagewall Yep, good eye. That's documented in the sample's README as a known issue.

**Pull request merged**